jvm-packages/xgboost4j/src/test/java/ml/dmlc/xgboost4j/java/BoosterImplTest.java

```diff
@@ -395,4 +395,15 @@ public void testTrainFromExistingModel() throws XGBoostError, IOException {
     TestCase.assertTrue(booster1error == booster2error);
     TestCase.assertTrue(tempBoosterError > booster2error);
   }
+
+  @Test(expected = XGBoostError.class)
+  public void testThrowErrorOnDisposedBooster() throws XGBoostError {
```

Comment:
we should make the test consistent what we fixed, i.e. we should test after reading the previous checkpoint, 

the previous booster and the current generated booster is `==`